packages/components/pie-modal/src/index.ts

```diff
@@ -15,17 +17,19 @@ import {
 export { type ModalProps, headingLevels };
 
 const componentSelector = 'pie-modal';
+const validPropertyValues = validPropertyValuesDecoratorFactory(componentSelector);
+const requiredProperty = requiredPropertyDecoratorFactory(componentSelector);
 
 export class PieModal extends RtlMixin(LitElement) {
     @property({ type: Boolean })
         isOpen = false;
 
     @property({ type: String })
-    @requiredProperty(componentSelector)
+    @requiredProperty()
```

Comment:
NIT: what if we just named it `required`?

Reply:
I initially wondered whether there might be anything else we need to decorate as `required`, but I'm not sure that's the case. We can probably get away with renaming it, yeah.